spec/satellite/models/user_spec.rb

```diff
@@ -42,7 +42,7 @@ def auth(options = {})
       user = User.find_or_create_with_omniauth(auth('info' => info))
       expect(user.name).to eq('Missy Wednesday')
       expect(user.email).to eq('missy@example.com')
-      expect(user.image_url).to eq('https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50')
+      expect(user.image).to eq('https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50')
```

Comment:
In my opinion `image` is not the best choice as it's not what we use in platform nor copro.